themes/classic/_dev/js/listing.js

```diff
@@ -230,7 +230,16 @@ function updateProductListDOM(data) {
   $(prestashop.themeSelectors.listing.listTop).replaceWith(
     data.rendered_products_top,
   );
-  $(prestashop.themeSelectors.listing.list).replaceWith(data.rendered_products);
+
+  const renderedProducts = $(data.rendered_products);
+  const productSelectors = $(prestashop.themeSelectors.listing.product, renderedProducts);
+
+  if (productSelectors.length > 0) {
+    productSelectors.removeClass().addClass($(prestashop.themeSelectors.listing.product).first().attr('class'));
```

Comment:
```suggestion
    productSelectors.removeClass().addClass($(prestashop.themeSelectors.listing.product).first().attr('class'));
```

Why `removeClass` has no parameters ?

Reply:
Used to remove all class from a jQuery element